torch/csrc/jit/codegen/cuda/partial_split_map.cpp

```diff
@@ -24,16 +24,12 @@ void PartialSplitMap::build(Fusion* fusion) {
         continue;
       }
       auto root_domain = split->in();
-      auto kir_root_domain =
-          gpu_lower->lowerValue(split->in())->as<IterDomain>();
       auto start_offset = split->startOffset();
       start_offset_map_.insert({root_domain, start_offset});
-      kir_start_offset_map_.insert(
-          {kir_root_domain, gpu_lower->lowerValue(start_offset)->as<Val>()});
+      kir_start_offset_map_.insert({root_domain, start_offset});
```

Comment:
Can you please remove `kir_start_offset_map_` itself? (Ditto for `kir_stop_offset_map_` as well)

Reply:
Just noticed they are removed in the subsequent PR.